client/rest-high-level/src/test/java/org/elasticsearch/client/RequestTests.java

```diff
@@ -1115,14 +1119,10 @@ private static void resizeTest(ResizeType resizeType, CheckedFunction<ResizeRequ
             if (randomBoolean()) {
                 randomAliases(createIndexRequest);
             }
-            if (randomBoolean()) {
-                setRandomWaitForActiveShards(createIndexRequest::waitForActiveShards, expectedParams);
-            }
+            setRandomWaitForActiveShards(createIndexRequest::waitForActiveShards, expectedParams);
             resizeRequest.setTargetIndex(createIndexRequest);
         } else {
-            if (randomBoolean()) {
-                setRandomWaitForActiveShards(resizeRequest::setWaitForActiveShards, expectedParams);
-            }
+            setRandomWaitForActiveShards(resizeRequest::setWaitForActiveShards, expectedParams);
```

Comment:
why these two changes? I meant to to randomly choose one of the two branches, but then still set the param only randomly.

Reply:
`if (randomBoolean())` is already wrapped in the `setRandom*`. If we keep it here, then `waitForRandomShards` is set to a random value only 25% of the time.
I assumed that it was an oversight...

```
private static void setRandomWaitForActiveShards(Consumer<ActiveShardCount> setter, Map<String, String> expectedParams) {
        if (randomBoolean()) {
            String waitForActiveShardsString;
            int waitForActiveShards = randomIntBetween(-1, 5);
            if (waitForActiveShards == -1) {
                waitForActiveShardsString = "all";
            } else {
                waitForActiveShardsString = String.valueOf(waitForActiveShards);
            }
            setter.accept(ActiveShardCount.parseString(waitForActiveShardsString));
            expectedParams.put("wait_for_active_shards", waitForActiveShardsString);
        }
    }
```
I do not think that we need

Reply:
oh right :) thanks